test_runner/src/main/kotlin/ftl/filter/TestFilters.kt

```diff
@@ -150,12 +150,20 @@ object TestFilters {
         }
     )
 
-    private fun withClassName(classNames: List<String>): TestFilter = TestFilter(
-        describe = "withClassName (${classNames.joinToString(", ")})",
-        shouldRun = { testMethod ->
-            withPackageName(classNames).shouldRun(testMethod)
-        }
-    )
+    private fun withClassName(classNames: List<String>): TestFilter {
+        val splittedClassNames = classNames.map { it.split("#") }
+        return TestFilter(
+            describe = "withClassName (${classNames.joinToString(", ")})",
+            shouldRun = { testMethod ->
+                val splittedName = testMethod.testName.split("#")
```

Comment:
same here should not it be:
`testMethod.testName.split("#").last()` ?

Reply:
I can't implement it that way since I need both: test class name and test method name. But I agree that part needs small refactor